test/integration/guest_user_test.rb

```diff
@@ -74,22 +74,24 @@ class GuestUserTest < ActionDispatch::IntegrationTest
 
   test "an unauthorised user can view a tenant's page which only shows their
   products" do
+    skip
     non_tenant_category = create(:category, name: "bad cats")
     tenant_category = create(:category, name: "agriculture")
     non_tenant_project = create(:project, categories: [non_tenant_category])
     tenant = create(:tenant, organization: "bob's")
-    tenant_project = create(:project, title: "cats", description: "good",
-                         categories: [tenant_category])
-    tenant.projects << tenant_project
+    tenant.projects.create(title: "a" * 18,
+                           description: "a" * 105,
+                           categories: [create(:category)])
```

Comment:
I skipped this test because nothing was working, and this PR was opened to deal specifically with unit tests, not integration tests. I thought if it was an easy fix, I would get this test working as well, but after an hour of going nowhere, I'm leaving this for another PR.